### ISSUE TITLE
metadata.rb format is invalid

### ISSUE THREAD

**Issue by misheska** [chef-cookbooks/yum-epel#38]
### Cookbook version
1.0.1

### Chef-client version
12.12.15

### Platform Details
CentOS 6

### Scenario:
When trying to re-publish this cookbook to an internal supermarket, the supermarket plugin says that the `metadata.rb` format is incorrect.

### Steps to Reproduce:
Publish with `knife supermarket share`:

```
/opt/chefdk/bin/knife supermarket share yum-epel "Other" --config <my_knife_rb> --supermarket-site <my_supermarket_site> --cookbook-path .
```

### Expected Result:
Cookbook publishes successfully without error.

### Actual Result:
`knife cookbook share` returns the following error:

```
/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.12.15/lib/chef/cookbook/metadata.rb:763:in `new_args_format': The dependency specification syntax you are using is no longer valid. You may not (Chef::Exceptions::ObsoleteDependencySyntax)
specify more than one version constraint for a particular cookbook.
Consult https://docs.chef.io/config_rb_metadata.html for the updated syntax.
 
Called by: depends 'yum', ">= 3.6", "< 5.0"
```

**Comment by tarak** [comment]
+1

**Comment by 4thAce** [comment]
+1

**Comment by tas50** [comment]
Shipped in 1.0.2
